tests/functional/all_daemons/group_vars/mons

```diff
@@ -1,6 +1,6 @@
 ---
-create_crush_tree: True
-crush_rule_config: True
+create_crush_tree: false
+crush_rule_config: false
```

Comment:
These two are making problems for the CI which doesn't have the HDD crush rule.

Reply:
they shouldn't be false